src/builder.rs

```diff
@@ -619,7 +619,7 @@ impl Builder {
         &self.outputs
     }
 
-    /// The net value of the bundle to be built. The value of all spends,
+    /// The net ZEC value of the bundle to be built. The value of all spends,
```

Comment:
Let's be even more explicit
`The net native (ZEC) value of the bundle to be built. The value of all spends,`